templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-restore-sh.yaml

```diff
@@ -23,6 +23,12 @@ contents:
       SNAPSHOT_FILE="$1"
     fi
     
+
+    ETCD_CONNSTRING=""
+    if [ "$2" != "" ]; then
```

Comment:
Can you add this to the example [1] otherwise lgtm

https://github.com/openshift/machine-config-operator/blob/28011239345ebeb0ba8de73c6deb142160511c75/templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-restore-sh.yaml#L12

Reply:
Right, fixed.

These are optional params, not sure if we want to specify that in the example string

Reply:
yeah, figure folks will read docs but if they see this it will give them hint to functionality, thanks.